server/config/nl_page/prop_titles.json

```diff
@@ -0,0 +1,18 @@
+{
+  "<-referenceSNPClusterID{typeOf:GeneticVariantGeneAssociation}->geneSymbol": {
+    "displayName": "associated gene",
+    "titleFormat": "The associated genes for {entity} are"
```

Comment:
(TODO for now but) do we need something for singular?

Reply:
added TODO